x/staking/types/validator.go

```diff
@@ -395,6 +386,7 @@ func (v Validator) RemoveDelShares(pool Pool, delShares sdk.Dec) (Validator, Poo
 // UNITS: tokens/delegator-shares
 func (v Validator) DelegatorShareExRate() sdk.Dec {
 	if v.DelegatorShares.IsZero() {
+		// TODO: When does this happen and is this semantically correct?
```

Comment:
The reason is when there are no delegations in an empty validator, the first delegations "set" the delegation exchange rate to `1` - let's update this comment to reflect this

Reply:
Ah of course - updated comment